web/frontend/src/components/do-command/get-client-by-type.ts

```diff
@@ -1,44 +1,57 @@
 import {
+  ArmClient,
+  BaseClient,
+  BoardClient,
+  CameraClient,
+  EncoderClient,
+  GantryClient,
+  GenericComponentClient,
+  GripperClient,
+  InputControllerClient,
+  MotionClient,
+  MotorClient,
+  MovementSensorClient,
+  NavigationClient,
+  PowerSensorClient,
   SensorClient,
+  ServoClient,
+  SlamClient,
+  VisionClient,
+  type Resource,
   type RobotClient,
-  CameraClient,
 } from '@viamrobotics/sdk';
 
 const CLIENT_TYPES = {
-  arm: 'armService',
-  base: 'baseService',
-  board: 'boardService',
-  encoder: 'encoderService',
-  gantry: 'gantryService',
-  generic: 'genericService',
-  gripper: 'gripperService',
-  input_controller: 'inputControllerService',
-  motion: 'motionService',
-  motor: 'motorService',
-  movement_sensor: 'movementSensorService',
-  navigation: 'navigationService',
-  power_sensor: 'powerSensorService',
-  sensors: 'sensorsService',
```

Comment:
Does the sensors service no longer exist? I was under the impression it still existed.

Reply:
there's no reified client for it other than the promise client. is anyone running a do command against a sensors service though?

Reply:
if a problem comes up with it, it wont be hard to get it working again!